notebook2/notebook/notebook.py

```diff
@@ -184,6 +184,7 @@ def start_pod(jupyter_token, image, name, user_id, user_data):
                 ],
                 name='default',
                 image=image,
+                env={'HAIL_TOKEN_FILE': '/user-jwt/jwt'},
```

Comment:
I think this needs to be a list...at least according to the spec

https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1Container.md